lib/tasks/vacuum.rake

```diff
@@ -0,0 +1,34 @@
+namespace :vacuum do
+  desc "Clean up old fiscal year proposals"
+  task old_proposals: :environment do
+    ok_to_act = false
+    if ENV["OK_TO_ACT"] && ENV["OK_TO_ACT"].match(/^[yt]/i)
+      ok_to_act = true
+    end
+    verbose = false
+    if ENV["VERBOSE"] && ENV["VERBOSE"].match(/^[yt]/i)
+      verbose = true
+    end
+    cleaner = ExpiredRecordCleaner.new(Time.zone.now, ok_to_act: ok_to_act, verbose: verbose)
+    if verbose
+      STDERR.puts "Pending proposals created before #{vacuum.fiscal_year_start}"
```

Comment:
when I ran this on c2-dev, I am getting `undefined local variable or method `vacuum' for main:Object` on this line